dc-current-branch.yml

```diff
@@ -0,0 +1,43 @@
+---
+version: "3.4"
+services:
+    database:
+        restart: always
+        image: postgres:12
+        container_name: herdbook-db
+        env_file:
+            - ./.docker/database-variables.env
+        environment:
+            - POSTGRES_HOST_AUTH_METHOD=trust
+        volumes:
+            - ./postgres-data:/var/lib/postgresql/data
+    herdbook-frontend-devel:
+        image: ghcr.io/nbisweden/herdbook_frontend:${TAG}
```

Comment:
It might be neat to use the `${TAG:?error-message}`-syntax to give a helpful error message if TAG isn't set properly!